tests/test_interpolation.py

```diff
@@ -14,6 +14,11 @@
 import scipy.sparse
 
 
+class SparseFirst(SparseFunction):
```

Comment:
why not making this  test-local ?

Reply:
Wasn't sure if was gonna need more tests for it in the future (I'm sure someone will come up with more bugs) so thought was better but happy to move it for now